installer/mac/README.md

```diff
@@ -60,10 +70,10 @@ The Xcode Command Line Tools package includes utilities for compiling code from
     ```bash
     cd ${CONDA_PREFIX}
     mkdir -p etc/conda/activate.d etc/conda/deactivate.d
-    echo "export OLD_LD_LIBRARY_PATH=\$LD_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
-    echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:\${CONDA_PREFIX}/lib" >> etc/conda/activate.d/env_vars.sh
-    echo "export LD_LIBRARY_PATH=\$OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
-    echo "unset OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
+    echo "export OLD_DYLD_LIBRARY_PATH=\$DYLD_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
```

Comment:
interesting, when did this change?

Reply:
I honestly don't know, but nobody in the workshop using the latest MacOS was able to get their installation to completely work. I just tested it on my laptop and it works with this fix though!